ui/src/components/layout/SideBar.vue

```diff
@@ -356,6 +356,11 @@
                 padding: 0 5px;
             }
 
+            /* Remove shadow/gradient overlay when sidebar is collapsed */
```

Comment:
This is not actually what we want, the better solution would be to remove this change and to amend line `382` in this file to be `10px`.

Is that something you're willing to tackle @Sehnya?

Thanks for the efforts so far! 🚀